fixtures/components/complex-types/column-layout/index.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+
+export interface ColumnLayoutProps {
+  columns: ColumnLayoutProps.Columns;
+  widths: ColumnLayoutProps.Widths;
+}
+
+export namespace ColumnLayoutProps {
+  export type Columns = 1 | 2 | 3 | 4;
+  export type Widths = 25 | '50%' | 100 | '33%';
```

Comment:
Added new test case on number-only and on mixed string+number union types.

We do not have a real use-case yet, but let's make sure it works anyway